src/Server/middleware/userRequiredMiddleware.ts

```diff
@@ -9,7 +9,6 @@ const USER_REQUIRED_ROUTES = [
   "/settings/purchases(.*)",
   "/settings/shipping",
   "/user/conversations(.*)",
-  "/collector-profile/saves2(.*)",
```

Comment:
Some details are being [discussed](https://artsy.slack.com/archives/C9SATFLUU/p1681497409074369)

Reply:
Decided to redirect to login form